Userland/Utilities/mktemp.cpp

```diff
@@ -78,8 +78,10 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
             target_directory = resolved_path.dirname();
             file_template = resolved_path.basename();
         } else {
-            char const* env_directory = getenv("TMPDIR");
-            target_directory = env_directory && *env_directory ? StringView { env_directory, strlen(env_directory) } : "/tmp"sv;
+            target_directory = "/tmp"sv;
+            auto const* env_directory = getenv("TMPDIR");
+            if (env_directory != nullptr)
```

Comment:
This doesn't preserve the same behaviour, a `TMPDIR` set to `""` will now result in `target_directory` being `""` (whereas it would've been `/tmp` before)